base/src/main/java/com/smartdevicelink/managers/ChoiceSetCompletionListener.java

```diff
@@ -0,0 +1,16 @@
+package com.smartdevicelink.managers;
+
+import com.smartdevicelink.managers.screen.choiceset.ChoiceCell;
+
+import java.util.HashSet;
+
+public interface ChoiceSetCompletionListener {
```

Comment:
Can you make this a part of BaseChoiceSetManager like AlertSoftButtonClearListener in BaseAlertManager?